documentation/bolt_project_directories.md

```diff
@@ -1,114 +1,91 @@
 # Project directories
 
-Bolt runs in the context of a project directory or a `Boltdir`. This directory
-contains all of the configuration, code, and data loaded by Bolt.
+There are two directory structures available to Bolt projects. You can run Bolt inside a
+local project directory and share that directory with others using a version control
+system like git, or you can embed the project directory inside an application's
+repo, and make Bolt available to that application.
 
-The project directory structure makes it easy to share Bolt code by committing
-the project directory to Git. You can then check different repositories of Bolt
-code into different directories in order to manage various applications.
+## Local project directories
 
-> **Note:** Bolt projects received some experimental updates in Bolt 2.8.0. For
-> more information, see [Bolt
-> projects](./experimental_features.md#bolt-projects). 
+To track and share management code in a dedicated repository, or to run and
+develop Bolt content locally, use a local project directory. Bolt recognizes
+any directory containing a `bolt-project.yaml` file as a Bolt project directory.
 
-## Types of project directories
-
-There are three types of project directories that you can use depending on how
-you're using Bolt.
-
-### Local project directory
-
-Bolt treats a directory containing a `bolt.yaml` or `bolt-project.yaml` file as
-a project directory.  Use this type of directory to track and share management
-code in a dedicated repository.
-
-> 🔩 **Tip:** You can use an existing control repo as a Bolt project directory
-  by adding a `bolt.yaml` file to it and configuring the `modulepath` to match
-  the `modulepath` in `environment.conf`.
-
-A project directory of this type has a structure like:
+A simple project directory looks like this:
 
 ```console
-project/
-├── Puppetfile
-├── bolt.yaml
+myproject/
 ├── bolt-project.yaml
-├── data
-│   └── common.yaml
 ├── inventory.yaml
-└── site-modules
-    └── project
-        ├── manifests
-        │   └── my_class.pp
-        ├── plans
-        │   ├── deploy.pp
-        │   └── diagnose.pp
-        └── tasks
-            ├── init.json
-            └── init.py
+├── plans
+│   ├── deploy.pp
+│   └── diagnose.pp
+└── tasks
+    ├── init.json
+    └── init.py
 ```
 
-### Embedded project directory
+As long as your `bolt-project.yaml` file contains a `name` field, Bolt loads
+your local Bolt content from the top level of your directory. If you're
+developing a module for the Puppet Forge, you can use a Puppet module
+directory structure. For more information, see [Module
+structure](module_structure.md). 
+
+
+> 🔩 **Tip:** You can use an existing control repo as a Bolt directory by adding
```

Comment:
```suggestion
> 🔩 **Tip:** You can use an existing Puppet control repo as a Bolt directory by adding
```

I think clarifying that this is for existing Puppet users might be helpful, since it can be a bit confusing what we're talking about if you've never used Puppet.